setup.cfg

```diff
@@ -1,2 +1,9 @@
 [bdist_wheel]
 universal=1
+
+[versioneer]
+VCS = git
```

Comment:
might want to add `style = pep440` here? Not sure what exactly it means but it seems to be in my other projects...

Reply:
pep440 is the default style, we can add it or not... doesn't really matter. it simply means that the version you get is pep440 compliant.